lib/wallaby/query/error_message.ex

```diff
@@ -47,6 +47,11 @@ defmodule Wallaby.Query.ErrorMessage do
     can't apply both filters correctly.
     """
   end
+  def message(_, {:get_number, size}) do
```

Comment:
We should pass the query in instead of size here. The query will have the result on it so we can get the size. The query will also have the requested item so we can change this message to something like:

```
The element at index #{query.get} is not available because #{result_count(query.result)} #{method(query)} #{result_expectation(query.result}
```